scrapper.py

```diff
@@ -26,38 +34,120 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        self.urls.append('https://magadanpravda.ru/' + article_bs.find('a')['href'])
```

Comment:
do you only add one URL from each seed URL? 
also, before adding a new link, it is better to check that this link has not yet been added as sources often place the same links over and over

Reply:
I checked, I do, indeed, collect unique URLs